pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools>=40.8.0", "wheel", "cython"]
```

Comment:
Is `cython` needed here?

Reply:
May be needed for this dependency: https://github.com/RalfG/python-wheels-manylinux-build

Reply:
As I understand this is necessary to play well with `pep517.build`. I haven't look into great details if this is _strickly_ necessary but when I looked at it I came to the conclusion that is the most simple and - it seems - the most future proof, even if it adds one more configuration file!

Reply:
Sure, I understand the necessity of the `pyproject.toml`, and for `wheel` and a minimum version for `setuptools`. It's specifically the `cython` requirement that I'm questioning. I think we shouldn't need this.